config/helper_test.go

```diff
@@ -0,0 +1,124 @@
+package config
+
+import (
+	"encoding/json"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+//nolint:gochecknoglobals
```

Comment:
I'm not sure if its added in the core already, but I disabled the globals linter in the plugins for tests, as in tests we do use globals often.